docs/semgrep-appsec-platform/notifications.md

```diff
@@ -4,25 +4,71 @@ append_help_link: true
 title: Alerts and notifications
 hide_title: true
 toc_max_heading_level: 4
-description: "Learn how to receive Slack or email alerts about findings and failures and how to integrate using webhooks."
+description: "Learn how to receive Slack, webhook, or email notifications about findings."
 tags:
     - Deployment
     - Semgrep AppSec Platform
 ---
 
+import RuleModes from "/src/components/reference/_rule-modes.md"
+
 # Alerts and notifications
 
-Users can receive notifications from Semgrep AppSec Platform in the following channels:
+You can receive notifications for Semgrep Code and Supply Chain findings in the following channels:
 
 - [Slack](/semgrep-appsec-platform/slack-notifications)
 - [Email](/semgrep-appsec-platform/email-notifications)
 - [Webhooks](/semgrep-appsec-platform/webhooks)
 
+
+Setting up notifications involves the following steps:
+
+1. Integrating the notification channel, such as Slack, with Semgrep.
+2. Customizing the conditions under which a notification is sent to that channel. Available conditions and how they are set up varies depending on the Semgrep product; see the following table.
+
+<table>
+<thead>
+<tr>
+<th>Channel</th>
+<th>Semgrep Code</th>
+<th>Semgrep Supply Chain</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td>Slack</td>
+<td rowspan="3">Customize through rule modes in [Policies page](/semgrep-code/policies)</td>
+<td>Limited customizability; configured by default to send notifications on <strong>reachable</strong> findings</td>
+</tr>
+<tr>
+<td>Email</td>
+<td>--</td>
```

Comment:
Do you think "not available" might be clearer here instead of the double dash?

Reply:
That is the TW answer and I will make this change, but in other parts of the docs I acceded to Sales to use `--` because it felt "softer" (in case you're wondering why some parts have `--` instead of "No" or "Not available".